pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -58,6 +58,13 @@ python3.pkgs.buildPythonApplication rec {
     # unsandboxed non-NixOS builds, see:
     # https://github.com/NixOS/nixpkgs/issues/86131#issuecomment-711051774
     ./boost-Do-not-add-system-paths-on-nix.patch
+
+    # https://github.com/mesonbuild/meson/pull/9841
+    # cross-compilation fix
+    (fetchpatch {
+      url = "https://github.com/mesonbuild/meson/pull/9841/commits/266e8acb5807b38a550cb5145cea0e19545a21d7.patch";
```

Comment:
Probably better to use direct commit link: `https://github.com/mesonbuild/meson/commit/266e8acb5807b38a550cb5145cea0e19545a21d7.patch`

The PR links sometimes cause issues.

Reply:
oops, yes I should. Didn't notice I used a PR link

Reply:
Fixed, thanks for the comment.